ios/BlindThresholdBls.m

```diff
@@ -1,14 +1,119 @@
 #import "BlindThresholdBls.h"
+#import "Headers/threshold.h"
+#import <React/RCTLog.h>
+#import <Foundation/Foundation.h>
+#import <stdlib.h>
+#import <stdint.h>
 
-
-@implementation BlindThresholdBls
+@implementation BlindThresholdBls {
```

Comment:
So if I'm understanding this correctly, we need a unique token/message for every phone number - and I don't think there is a lock right now? Would you would potentially want to store these for each request in a data structure with the phone hash as the identifier? Could also add a timestamp too to know when to clean up

Reply:
Correct, both the java + obj. c versions assume the caller is calling blind then unblind in the right order. No support for multiple outstanding requests yet.